server/database.js

```diff
@@ -188,6 +189,19 @@ class Database {
                 }
             };
         } else if (dbConfig.type === "mariadb") {
+            if (!/^\w+$/.test(dbConfig.dbName)) {
+                throw Error("Invalid Database name");
```

Comment:
I think this error message could be more self-explainatory.

```suggestion
                throw Error("Invalid database name. A database name can only consist of letters, numbers and underscores");
```